pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -155,7 +159,10 @@ func (l *libcontainerAdapter) newManager(cgroups *libcontainerconfigs.Cgroup, pa
 			panic("systemd cgroup manager not available")
 		}
 		if libcontainercgroups.IsCgroup2UnifiedMode() {
-			return cgroupsystemd.NewUnifiedManager(cgroups, paths["memory"], false), nil
+			return cgroupsystemd.NewUnifiedManager(cgroups, paths["memory"], l.rootless), nil
+		}
+		if l.rootless {
```

Comment:
Should kubefeatures.Rootless be checked here ?

Reply:
The feature gate is checked in `NewCgroupManager()`: https://github.com/kubernetes/kubernetes/pull/92863/files#diff-fa76bb6ae2d9b4bb3d023737fe5e6029R204

Reply:
thanks